<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>centralWidget</class>
 <widget class="QWidget" name="centralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
  font-family: Roboto;
  font-size: 16px;
}
QPushButton {
  font-size: 16px;
  padding: 8px;
}
QLabel {
  font-size: 16px;
  color: white;
}

#loginForm_2 QPushButton {
  font-size: 24px;
  padding: 12px
}

#centralWidget, QMessageBox {
  background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 #004563, stop:1 #48a1b7);
}
#header, #welcomeHeader, #replaceHeader, #addHeader, #addWiFiHeader, #updateWiFiHeader, #manageWiFiHeader, #systemSettingsHeader {
  margin: 20px;
  font-size: 35px;
}
#welcomeSubHeader {
  font-size: 25px;
}
#addForm QLabel, #replaceForm QLabel, #addWiFiForm QLabel {
  padding-right: 8px;
}
#addForm QPushButton, #replaceForm QPushButton, #addWiFiForm QPushButton {
  margin-top: 8px;
}

#welcomeUsername, #welcomeVersion, #loginVersion, #loginSelectedServerText,
#replaceControllerSerialPortsText, #addControllerSerialPortsText {
  color: #ccc;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTextEdit" name="monitorTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Roboto'; font-size:16px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16px; font-style:italic;&quot;&gt;Connect a controller, refresh the ports and press play...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comPortsComboBox">
       <property name="toolTip">
        <string>Select the port to read output from</string>
       </property>
       <item>
        <property name="text">
         <string>Port 1 (COM3)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Port 2 (/dev/ttyUSB0)</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateComButton">
       <property name="toolTip">
        <string>Update COM ports</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="system-reboot"/>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startStopButton">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Play or pause reading the serial output</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="media-playback-start"/>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="baudRateComboBox">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>512000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="encodingComboBox">
       <item>
        <property name="text">
         <string>8N1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7N1</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="toolTip">
        <string>Close this window</string>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
